specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -3453,6 +3457,16 @@
           "externalDocs": {
             "url": "https://docs.microsoft.com/azure/search/search-indexer-field-mappings"
           }
+        },
+		"outputFieldMappings": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FieldMapping"
+          },
+          "description": "Output field mappings are applied after enrichment and immediately before indexing.",
```

Comment:
@jeji1101 We don't really define the term "enrichment" here, so maybe it would help to have more context. How about:

"Applicable only to indexers with an associated skillset. Output field mappings are applied after documents are enriched by the skillset, immediately before indexing."